code/game/objects/items/devices/flashlight.dm

```diff
@@ -147,14 +150,85 @@
 	icon_state = "penlight"
 	item_state = ""
+	flags_equip_slot = SLOT_WAIST|SLOT_EAR|SLOT_SUIT_STORE
 	flags_atom = FPRINT|CONDUCT
 	light_range = 2
 	w_class = SIZE_TINY
+	throw_speed = SPEED_VERY_FAST
+	throw_range = 15
+	matter = list("metal" = 10,"glass" = 5)
 	raillight_compatible = 0
 
+/obj/item/device/flashlight/pen/attack(mob/living/carbon/human/being as mob, mob/living/user as mob)
+	add_fingerprint(user)
+	if(user.a_intent == INTENT_HELP)
+		if(on && user.zone_selected == "eyes")
+			if(!ishuman_strict(being)) //robots and aliens are unaffected
+				return
+			var/reaction = "try to watch closely, but you see no difference in [being.p_their()] eyes' reactions" //Shouldn't ever happen anyways
+			if(isnull(being.internal_organs_by_name))
+				reaction = "discover that indeed [being.p_they()] have nothing to be checked"
+				return // they have no organs somehow
+			if(being == user) //they're using it on themselves
+				being.flash_eyes()
+				being.visible_message(SPAN_NOTICE("[being] directs [src] to [being.p_their()] eyes."), \
+							SPAN_NOTICE("You wave the light in front of your eyes! Wow, that's trippy!"))
+				return
+			if(being.stat == DEAD || (being.status_flags&FAKEDEATH))
+				reaction = "conclude that [being.p_their()] eyes are completely lifeless, [being.p_they()] must have passed away"
+			else
+				var/datum/internal_organ/eyes/eyes = being.internal_organs_by_name["eyes"]
+				var/datum/internal_organ/brain/brain = being.internal_organs_by_name["brain"]
+				if(skillcheck(user, SKILL_MEDICAL, SKILL_MEDICAL_MEDIC))
+					if(eyes)
+						switch(eyes.organ_status)
+							if(ORGAN_LITTLE_BRUISED)
+								being.flash_eyes()
+								reaction = "notice that [being.p_their()] eyes are <font color='yellow'>reacting to the light</font>, but [being.p_their()] pupils seen to <font color='yellow'>react sluggishly and with small delays</font>, [being.p_their()] vision is probably <font color='yellow'>a little impaired</font>"
+							if(ORGAN_BRUISED)
+								being.flash_eyes()
+								reaction = "observe that [being.p_their()] eyes are <font color='orange'>unrealiably reacting to the light</font>, with [being.p_their()] pupils <font color='orange'>reacting very sluggishly and with noticeable delays</font>, it is probable that [being.p_their()] vision is <font color='orange'>remarkably impaired</font>"
+							if(ORGAN_BROKEN)
+								reaction = "notice that [being.p_their()] eyes are <font color='red'>not reacting to the light</font>, and the pupils of both eyes are <font color='red'>not constricting with the light</font> shine at all, [being.p_they()] is probably <font color='red'>blind</font>"
+							else
+								being.flash_eyes()
+								reaction = "perceive that [being.p_their()] eyes and pupils are <font color='green'>normally reacting to the light</font>, [being.p_they()] is probably<font color='green'> seeing without problems</font>"
+					if(brain)
```

Comment:
final thing, check if the reaction exists here and if it does just add ", You also". There is a lot of copypaste code here for essentially same thing but adds ". You also" on top of that, reaction is never zero, so I dont see the purpose of the checks below

Reply:
so add `if (reaction) reaction += ". You also"` right after if(brain), and clean up the text below, I believe in you, mark as ready when done